docs/source/protocols/433.92/alarm/secudo.rst

```diff
@@ -0,0 +1,72 @@
+.. |yes| image:: ../../../images/yes.png
+.. |no| image:: ../../../images/no.png
+
+.. role:: underline
+   :class: underline
+
++------------------+-------------+
+| **Feature**      | **Support** |
++------------------+-------------+
+| Sending          | |no|        |
++------------------+-------------+
+| Receiving        | |yes|       |
++------------------+-------------+
+| Config           | |yes|       |
++------------------+-------------+
+
+.. rubric:: Supported Brands
+
++---------------------------+------------------+
+| **Brand**                 | **Protocol**     |
++---------------------------+------------------+
+| secudo (formerly FlammEx) | secudo           |
++---------------------------+------------------+
+
+.. rubric:: Config
+
+.. code-block:: json
+   :linenos:
+
+   {
+     "devices": {
+       "alarm": {
+         "protocol": [ "secudo" ],
+         "id": 493,
+         "state": "alarm"
+       }
+     },
+     "gui": {
+       "alarm": {
+         "name": "Smoke Sensor",
+         "group": [ "Sleeping" ],
+         "media": [ "all" ]
+       }
+     }
+   }
+
++------------------+-----------------+
+| **Option**       | **Value**       |
++------------------+-----------------+
+| id               | 0 - 1023        |
++------------------+-----------------+
+| state            | alarm           |
++------------------+-----------------+
+
+.. rubric:: Protocol
+
+This protocol sends 26 pulses like this
+
+.. code-block:: console
+
+   248 341 558 651 279 310 620 310 620 651 279 310 620 310 620 310 589 341 589 651 279 651 279 310 620 10633
+
+The pairs of the 2nd and 3rd, the 4th and 5th etc. pulses are either long-short or short-long.
```

Comment:
Can you describe the protocol more like this:
https://github.com/pilight/pilight/blob/development/docs/source/protocols/433.92/contact/kaku.rst